javalib/src/main/scala/java/util/LinkedList.scala

```diff
@@ -16,17 +16,20 @@ class LinkedList[E]() extends AbstractSequentialList[E]
   private var head: Node[E] = null
   private var last: Node[E] = null
 
-  private var _size = 0
+  /**Inner size is represented with a Double to satisfy Collection size method requirement:
+   * If this collection contains more than Integer.MAX_VALUE elements, returns Integer.MAX_VALUE.
+   */
+  private var _size: Double = 0
```

Comment:
Don't make it a ScalaDoc comment. So replace the second `*` by a space.

Also, cut the lines at 80 characters.